src/en/getting-started.md

```diff
@@ -24,65 +21,116 @@ Both the above are provided with Ubuntu 16.04LTS.
 
 Run the following commands to install the required software:
 
-```bash
-sudo add-apt-repository ppa:juju/devel
+```no-highlight
 sudo apt update
-sudo apt install juju zfsutils-linux lxd
-newgrp -
+sudo apt install juju-2.0 zfsutils-linux
 ```
 
-!!! Note: The `newgrp` command should only be necessary if package 'lxd' got
-installed (it may be already installed). This command will update your group
-memberships but will also start a new shell.
-
-
-## Prepare ZFS
+## Initialise LXD
 
-Using ZFS we can create sparse backing-storage for any of the containers which
-LXD creates for Juju. You can create this storage anywhere (e.g. the fastest
-drive you have). This example creates a 32G file to use:
+In order to use LXD, your user must be in the 'lxd' group. All system users are
+automatically added to this group, but you may need to refresh the current 
+session. You can confirm your user is part of this group by running the command:
 
 ```bash
-sudo mkdir /var/lib/zfs
-sudo truncate -s 32G /var/lib/zfs/lxd.img
-sudo zpool create lxd /var/lib/zfs/lxd.img
+groups
 ```
+This should indicate the user is a member of the lxd group, amongst others:
 
-As this is sparse storage, it won't actually take up disk space until it is 
-actually being used. You can check the file has been added to a ZFS pool with 
-the following command:
+```bash 
+lxd adm cdrom sudo dip plugdev lpadmin sambashare ubuntu
```

Comment:
Listing groups can be a cause of confusion/frustration if the reader's user looks different.

s/bash/no-highlight if we keep this kind of output.